src/Components/Color/ContrastChecker.jsx

```diff
@@ -5,20 +5,23 @@ export default function ContrastChecker({ hex, contrastText }) {
 
   useEffect(() => {
     async function postFetch() {
-      fetch("https://www.aremycolorsaccessible.com/api/are-they", {
-        method: "POST",
-        body: JSON.stringify({ colors: [hex, contrastText] }),
-        headers: {
-          "Content-Type": "application/json",
-        },
-      })
-        .then((response) => response.json())
-        .then((json) => {
-          setResult(json.overall);
-        });
+      const response = await fetch(
+        "https://www.aremycolorsaccessible.com/api/are-they",
+        {
+          method: "POST",
+          body: JSON.stringify({ colors: [contrastText, hex] }),
+          headers: {
+            "Content-Type": "application/json",
+          },
+        }
+      );
+      const contrastData = await response.json();
+      const contrastScore = contrastData.overall;
+
+      setResult(contrastScore);
```

Comment:
you could think of including a response error validation like `if (!response.ok) {
        throw new Error("Failed to fetch data!");` or try and catch